modules/multiwallet.js

```diff
@@ -0,0 +1,33 @@
+const app   = require('electron').app;
+const spawn = require('buffered-spawn');
+const path  = require('path');
+const log   = require('electron-log');
+
+function getWalletsPath() {
+
+  const platform = process.platform
+    .replace('freebsd', 'linux')
+    .replace('sunos',   'linux');
+
+  if (platform == 'linux') {
+    return path.join(app.getPath('home'), '.particl');
+  } else {
+    return app.getPath('userData');
+  }
+}
+
+exports.get = function () {
+  return new Promise((resolve, reject) => {
+
+    spawn('ls', [ getWalletsPath() ]).then(files => {
```

Comment:
Does this work on windows? 'ls' vs 'dir'

Reply:
good catch, thanks !

Reply:
I disabled multiwallet in this PR as it will be done in another one.